### DISCUSSION TITLE
Vercel Integration

### DISCUSSION THREAD

**Discussion by NuroDev** [planetscale/discussion#46] [Documentation] [answered]
### What?
The current documentation, while very helpful, proves that while it may technically be possible to deploy to [Vercel](https://vercel.com), it cannot be done so currently "easily". However, [Vercel](https://vercel.com) offers a very nice [integrations](https://vercel.com/dashboard/integrations) system which allows you to easily connect 3rd party system/applications to a deployment. 

With PlanetScale being primarily targeted towards being a "serverless database platform" it would be amazing if I was able to connect these 2 platforms with a simple integration.

**Comment by quetzalliwrites** [comment, marked as answer]
Hey there @NuroDev! Thank you for bringing this up, as we are definitely excited to keep adding to the Docs. *Deploying to Vercel* is a great tutorial we would love to add, and we'll be adding more info on this soon. 😁🪐

**Reply by shaunchurch** [reply]
Is this even possible in a serverless environment while requiring the CLI to be used as a proxy? Excited to see this written up.

**Reply by quetzalliwrites** [reply]
Yes! Good news everyone! A further update...

You asked for a Vercel integration and now [here it is](https://vercel.com/integrations/planetscale?utm_partner=planetscale)!

To get started, follow this **PlanetScale tutorial**:
https://docs.planetscale.com/tutorial/deploy-to-vercel

**Reply by NuroDev** [reply]
Amazing! Thank you! 👏🏻 🎉

**Reply by arist0tl3** [reply]
I know this doesn't solve the _exact_ issue, but maybe consider cross posting to the connection string feature request thread as that's where I've been checking for a solution like this one.

**Comment by bigint** [comment]
I'm waiting for Prisma in Vercel support 🤞🏻

**Reply by janpio** [reply]
Prisma is waiting for that as well :D 

It will be possible via https://github.com/planetscale/beta/discussions/15

**Comment by techandmedia** [comment]
The planetscale integration return 404 for me. Is it not available in hobby plan?

=== updated
Nervermind, it shows up now
